requirements.master.txt

```diff
@@ -1 +1,3 @@
 tx-manager>=0.2,<0.3
```

Comment:
Would it be better to have this pull from Github also? If we need a particular version we could specify a tag.